apps/css-workshop/src/components/Button.astro

```diff
@@ -0,0 +1,22 @@
+---
+type Props = {
+  variant?: 'high-visibility' | 'borderless' | 'cta';
+  size?: 'small' | 'large';
+  active?: 'true';
```

Comment:
Should this be a boolean?

Reply:
I left this prop as a string to keep things consistent with how `data-iui-active` was used and to avoid having to perform a check in the component when setting `data-iui-active`.  Making active a boolean does make using the prop easier when using the component, so I have made a [commit](https://github.com/iTwin/iTwinUI/pull/1987/commits/fab6716cea1ea8fd5e11080d3a05f5a433db34eb) with this change. Let me know what you think.

Reply:
Great, I feel the boolean prop is better and more consistent with how we usually do it. We also did it similarly in https://github.com/iTwin/iTwinUI/pull/1970.

https://github.com/iTwin/iTwinUI/blob/d635a7cb09b04eb5d02f1fc648ffb23ac846065c/apps/css-workshop/src/components/Checkbox.astro#L36